plat/hisilicon/hikey960/include/platform_def.h

```diff
@@ -104,7 +104,7 @@
 #define NS_BL1U_SIZE			(0x00100000)
 #define NS_BL1U_LIMIT			(NS_BL1U_BASE + NS_BL1U_SIZE)
 
-#define HIKEY960_NS_IMAGE_OFFSET	(0x1AC18000)	/* offset in l-loader */
+#define HIKEY960_NS_IMAGE_OFFSET	(0x1AC28000)	/* offset in l-loader */
```

Comment:
@hzhuang1, you are correct! The `l-loader` code will need a change to place the `NS_BL1U` (UEFI) `0x1000` bytes farther down.

This 96KB offset should be changed to 160KB, similarly to the HiKey: https://github.com/96boards-hikey/l-loader/blob/testing/hikey960_v1.2/gen_loader_hikey960.py#L12

With this change to `l-loader`:
```diff
diff --git a/gen_loader_hikey960.py b/gen_loader_hikey960.py
index 5c5e8fe..b230ad3 100755
--- a/gen_loader_hikey960.py
+++ b/gen_loader_hikey960.py
@@ -9,7 +9,7 @@ import string
 
 class generator(object):
     block_size = 512
-    ns_bl1u_lba = 192  # 96KB
+    ns_bl1u_lba = 320  # 160KB
 
     # set in self.add()
     idx = 0
```

This TBB change boots the recovery.